testing/jormungandr-integration-tests/src/common/jormungandr/mod.rs

```diff
@@ -8,7 +8,7 @@ pub use benchmark::storage_loading_benchmark_from_log;
 pub use configuration_builder::ConfigurationBuilder;
 pub use logger::{JormungandrLogger, LogEntry};
 pub use process::*;
-pub use rest::JormungandrRest;
+pub use rest::{JormungandrRest,RestError};
```

Comment:
aerate your imports, a space between `,` and the following character
```suggestion
pub use rest::{JormungandrRest, RestError};
```

Reply:
Apologies for that and other fmt issues. i was running `cargo fmt` in testing directory not in the root folder